packages/gestalt/src/ActivationCard.js

```diff
@@ -0,0 +1,147 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from './Box.js';
+import Heading from './Heading.js';
+import Icon from './Icon.js';
+import IconButton from './IconButton.js';
+import Button from './Button.js';
+import Text from './Text.js';
+import { useColorScheme } from './contexts/ColorScheme.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+import styles from './ActivationCard.css';
+
+type LinkData = {|
+  accessibilityLabel?: string,
+  href: string,
+  label: string,
+  onClick?: AbstractEventHandler<
+    | SyntheticMouseEvent<HTMLButtonElement>
+    | SyntheticMouseEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLButtonElement>
+  >,
+|};
+
+type Props = {|
+  dismissButton?: {|
+    accessibilityLabel: string,
+    onDismiss: () => void,
+  |},
+  message: string,
+  linkData?: LinkData,
+  status: 'notStarted' | 'pending' | 'needsAttention' | 'complete',
+  statusMessage: string,
+  title?: string,
+|};
+
+const STATUS_ICONS = {
+  notStarted: undefined,
+  pending: { symbol: 'clock', color: 'gray' },
+  needsAttention: { symbol: 'workflow-status-problem', color: 'red' },
+  complete: { symbol: 'check-circle', color: 'green' },
+};
+
+const ActivationCardLink = ({ data }: {| data: LinkData |}): Node => {
+  const { accessibilityLabel, href, label, onClick } = data;
+
+  return (
+    <Box
+      alignItems="center"
+      paddingX={1}
+      marginEnd="auto"
+      marginStart="auto"
+      mdMarginEnd={0}
+      mdMarginTop={8}
+      mdMarginBottom="auto"
+      rounding="pill"
+      flex="grow"
+    >
+      <Button
+        accessibilityLabel={accessibilityLabel}
+        color="gray"
+        href={href}
+        onClick={onClick}
+        role="link"
+        size="lg"
+        text={label}
+      />
+    </Box>
+  );
+};
+
+export default function ActivationCard({
+  dismissButton,
+  message,
+  linkData,
+  status,
+  statusMessage,
+  title,
+}: Props): Node {
+  const isStarted = status !== 'notStarted';
+  const icon = STATUS_ICONS[status];
+
+  return (
+    <Box flex="grow" borderSize="sm" rounding={4} padding={6} maxWidth={400}>
+      <Box display="flex" alignContent="center">
```

Comment:
@norrise120 Don't we want

![image](https://user-images.githubusercontent.com/127199/94977734-fd0a0b00-04ce-11eb-85bc-b606ce8f6d01.png)

instead of

![image](https://user-images.githubusercontent.com/127199/94977746-098e6380-04cf-11eb-9748-c3a42d55a051.png)

If so, we should set a flex grow on a `<Box>` which wraps the title